app.js

```diff
@@ -60,27 +57,22 @@ try {
 	require.resolve('./config/config');
 } catch (err) {
 	if (err.code !== 'MODULE_NOT_FOUND') throw err; // should never happen
-
-	// Copy it over synchronously from config-example.js since it's needed before we can start the server
-	console.log("config.js doesn't exist - creating one with default settings...");
-	fs.writeFileSync(path.resolve(__dirname, 'config/config.js'),
-		fs.readFileSync(path.resolve(__dirname, 'config/config-example.js'))
-	);
-} finally {
-	global.Config = require('./config/config');
+	throw new Error('config.js does not exist; run node pokemon-showdown to set up the default config file before launching Pokemon Showdown again.');
 }
 
+global.Config = require('./config/config');
+
 if (Config.watchconfig) {
 	let configPath = require.resolve('./config/config');
-	fs.watchFile(configPath, (curr, prev) => {
+	require('fs').watchFile(configPath, (curr, prev) => {
```

Comment:
Let me put it this way, I'm fine with using `require()` inline if it's the only place the module is required in the entire codebase, and it's an optional module. Otherwise, put it at the top.